README.md

```diff
@@ -400,10 +400,14 @@ Meeting agenda are hosted and archived in the [meetings repo](https://github.com
 #### Errata from the ACSAC Paper
 
 We discovered a typo in Figure 5 of the published ACSAC paper. The final interaction
-between the Tenant and Cloud Verifier showed an HMAC of the node's ID using the key
```

Comment:
i think this should stay node.  since it's about the acsac paper that we aren't updating